fontc/src/main.rs

```diff
@@ -158,7 +159,7 @@ fn glyphs_deleted<'a>(current_inputs: &Input, prev_inputs: &'a Input) -> HashSet
 
 fn add_glyph_work(
     context: &mut CompileContext,
-    work: &mut Vec<Box<dyn Work<()>>>,
+    work: &mut Vec<Box<dyn Work>>,
```

Comment:
not really about this PR, but this fn feels like it could maybe be a method on `CompileContext`?

Reply:
I have to think a little about where the actual task/dep data lives ... that might make sense indeed.